Source/MaestroAdvance.cpp

```diff
@@ -10,6 +10,12 @@ Maestro::AdvanceTimeStep (bool is_initIter) {
     // timer for profiling
     BL_PROFILE_VAR("Maestro::AdvanceTimeStep()",AdvanceTimeStep);
 
+// #ifdef AMREX_USE_CUDA
```

Comment:
Can you go through and clean up these commented-out launch region flags?

Also do we need to do the not_launched check? Can't we just set true or false in the places where we want it to be true or false, regardless of what it previously was?

Reply:
The not_launched checked is in the case that a function with a launch region calls another function that also has a launch region. It's to stop the function called by the other from closing the launch region when you still want it to be running in the higher up function.

Reply:
Got rid of commented out launch region flags in 0ab9df3